packages/tests/server/caller.test.ts

```diff
@@ -19,7 +19,7 @@ test('undefined input query', async () => {
   const caller = router.createCaller({});
   const result = await caller.hello();
 
-  expectTypeOf<string>(result);
```

Comment:
It should still be a `string` here @sachinraja, right?

https://github.com/trpc/trpc/actions/runs/4217420038/jobs/7321186668#step:9:126

Reply:
ah right, leftover from my testing